edalize/templates/symbiflow/symbiflow-vpr-makefile.j2

```diff
@@ -28,22 +39,22 @@ SDC_OPTS = -s ${SDC}
 all: ${TOP}.bit
 
 ${TOP}.eblif:
-	symbiflow_synth -t ${TOP} -v ${VERILOG} -d ${BITSTREAM_DEVICE} -p ${PARTNAME} ${XDC_OPTS}
+	${SOURCE_OPTS} && ${TOOLCHAIN_PREFIX}synth -t ${TOP} -v ${VERILOG} -d ${BITSTREAM_DEVICE} ${PARTNAME_OPT} ${XDC_OPTS}
```

Comment:
Drop the `&&` here (and below), otherwise this fails if environment_script is empty because the lines below will start with `&&` which causes syntax errors

Reply:
Fixed